frontend/src/pages/SearchPage.js

```diff
@@ -1,29 +1,38 @@
-import SpotItem from "../component/SpotItem";
 import styled from "styled-components/macro";
 import {FormControl, InputLabel, makeStyles, Select} from "@material-ui/core";
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import SpotList from "../component/SpotList";
+import axios from "axios";
 
 
 
 
-export default function SearchPage({spots}){
-
-
+export default function SearchPage(){
 
         const classes = useStyles();
         const [filterCountry, setFilterCountry] = React.useState({
             name: '',
         });
 
+        const [surfSpots, setSurfSpots] = useState([])
+
         const handleChangeCountry = (event) => {
             const name = event.target.name;
             setFilterCountry({
                 ...filterCountry,
                 [name]: event.target.value,
             });
+
         };
 
+    useEffect(() => {
+        axios.get("/api/surfspots/all")
+            .then((response) => response.data)
+            .then(data => setSurfSpots(data))
```

Comment:
```suggestion
            .then(setSurfSpots)
```

Reply:
True :) I always forget the short way :) THX